src/software/ai/hl/stp/skill/shoot/shoot_skill_fsm.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include "software/ai/hl/stp/skill/pivot_kick/pivot_kick_skill_fsm.h"
+#include "software/ai/hl/stp/skill/skill_fsm.h"
+
+struct ShootSkillFSM
```

Comment:
docs

Reply:
typically only add doc for the Skill/Tactic and not the FSM